git/keep-feature-branches-up-to-date.md

```diff
@@ -0,0 +1,25 @@
+# How To: Keep Feature Branches Up To Date
+
+If commits come into `upstream/master` after your branch off, you will need to keep your branch up to date with `upstream/master` before you PR. To do this, I recommend the slightly more challenging, yet more simple `rebase` method.
```

Comment:
What do you think about saying "after you branch off" or "after you create your branch" instead of "after your branch off"? Think it might be a little clearer. Also, "slightly more challenging, yet more simple" than what? On first read, this makes me think I've missed a sentence about another method that's less challenging and less simple.